MonoGame.Framework/Graphics/States/BlendState.cs

```diff
@@ -241,7 +241,20 @@ static BlendState()
 	    internal BlendState Clone()
 	    {
 	        return new BlendState(this);
-	    }
-	}
+	    }
+
+        protected override void Dispose(bool disposing)
+        {
+            if (!IsDisposed)
+            {
+                for (int i = 0; i < _targetBlendState.Length; ++i)
+                    _targetBlendState[i] = null;
+#if DIRECTX
+                PlatformDispose();
```

Comment:
Maybe you can declare this as a partial function... so that if it doesn't exist then it is ok?  Better than an `#if` I think.